### ISSUE TITLE
Searching for the same Python object/anchor does not reload page but just gets stuck on the search preview

### ISSUE THREAD

**Issue by jgarte** [jbms/sphinx-immaterial#200]
This is related to #198.

@jbms Thanks for the fix!

There is one edge case which is when you jump to the Python object and then try to search for the exact same Python object again, the search gets stuck and doesn't allow you to click.

I can imagine it working this way or reloading the same page with the same anchor.

What do you think?

**Comment by jbms** [comment]
I filed an upstream issue in mkdocs-material for this: https://github.com/squidfunk/mkdocs-material/issues/4829

**Comment by jbms** [comment]
This is now fixed in upstream, we just need to pull in those changes.

**Comment by 2bndy5** [comment]
I looked at that recently. Looks like upstream had a major version bump since our last update...

**Comment by jbms** [comment]
> I looked at that recently. Looks like upstream had a major version bump since our last update...

Yes, definitely non-trivial this time around.  I created https://github.com/jbms/sphinx-immaterial/issues/203 to track.

**Comment by jbms** [comment]
This is now fixed.
